templates/opps-list-widget.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This template is used to display the list of flexible volunteer opportunities
+ * for the widget.
+ *
+ * To adjust this template copy it into your current theme within a folder called "wivm".
+ */
+
+// Global variables that are set in widget/class-widget.php via the widget settings in admin 
```

Comment:
@jg314 @caseynix 

Yep, the globals are exactly for that reason - so that we could use the template loader class. I haven't found a way around using the global variables yet if using the template loader - since essentially what the loader does is locates the template and uses the WordPress load_template() function to require the file where the template loader was called. Unfortunately, the load_template() function only takes 2 arguments: the template file and require_once. There is no way to pass our widget options to our template using this.

I am still going to continue looking into this to see if I can come up with a solution. Do you guys have any thoughts?